components/about/SiteTimeline.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <v-timeline dense class="timeline-widget">
+      <v-timeline-item
+        v-for="item in items"
+        :key="item.date"
+        fill-dot
+        small
+        color="#96DCE8"
+      >
+      <v-container>
+        <v-row>
+            <v-col :cols="$vuetify.breakpoint.mobile ? 12 : 4">
```

Comment:
should we accounting for a middle tier on this? ipads and such?

Reply:
Perhaps, but we don't currently have a mock for a middle tier, so that's out of scope. Though, one could surely be made.

Reply:
Sounds like a good follow up design for later 👌